### PR TITLE
docs(recipes): add ssr recipe [fixes #54]

### LINKED ISSUE #54: recipe: server-side rendering
Documentation Is:

- [ ] Missing
- [x] Needed
- [ ] Confusing
- [ ] Not Sure?

### Please Explain in Detail...

Our server-side rendered app uses webpack to build bundles for both client and server (via multicompiler). Currently we're using https://github.com/60frames/webpack-hot-server-middleware for smooth DX. Code is being hot-reloaded on both client and server. Is this something that can be achieved using webpack-plugin-serve?

### PR DESCRIPTION
**Pull request by smashercosmo**
This PR contains:

- [ ] bugfix
- [ ] feature
- [ ] refactor
- [ ] tests
- [x] documentation
- [ ] metadata

### Breaking Changes?

- [ ] yes
- [x] no

### Please Describe Your Changes
SSR example added [fixes #54]

### CODE REVIEW

File: recipes/react-ssr/server/index.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+const statik = require('koa-static');
+
+const DIST_DIR = path.resolve(__dirname, '..', 'dist');
+const renderer = require(path.resolve(DIST_DIR, 'server.js'));
+
+const app = new Koa();
+app.use(renderer);
+app.use(statik(DIST_DIR));
+
+app.listen(3000, () => {
```
Comment (PlayMa256) [inline recipes/react-ssr/server/index.js]: won't it collide with the "client" server? Both running at 3000.

Comment (smashercosmo) [reply recipes/react-ssr/server/index.js]: This is production server

Comment (PlayMa256) [reply recipes/react-ssr/server/index.js]: oh, nvm.

**Comment by smashercosmo** [comment]
TODO:
1. Better README
2. To make DX even better webpack-plugin-serve should be able to suspend until bundle is ready. With webpack-dev-middleware it was possible with 'serverRenderer' option.

**Comment by smashercosmo** [comment]
TODO:
1. Better README
2. To make DX even better webpack-plugin-serve should be able to suspend until bundle is ready. With webpack-dev-middleware it was possible with 'serverRenderer' option.

**Comment by shellscape** [comment]
> To make DX even better webpack-plugin-serve should be able to suspend until bundle is ready. With webpack-dev-middleware it was possible with 'serverRenderer' option.

If you have the time, please open a new feature issue, and we can work on making that happen!

**Comment by shellscape** [comment]
OK @sibelius has given us the thumbs up offline. thanks @smashercosmo

**Comment by smashercosmo** [comment]
Awesome) Thx for amazing tool.

**Pull request merged**